arcgis-ios-sdk-samples/Scenes/Animate 3D graphic/PlaneStatsViewController.swift

```diff
@@ -16,10 +16,57 @@
 import UIKit
 
 class PlaneStatsViewController: UITableViewController {
+    var frame: Frame? {
+        didSet {
+            updateUI()
+        }
+    }
+    let measurementFormatter: MeasurementFormatter = {
+        let formatter = MeasurementFormatter()
+        formatter.numberFormatter.maximumFractionDigits = 0
+        formatter.unitOptions = .providedUnit
+        return formatter
+    }()
     
-    @IBOutlet var altitudeLabel: UILabel!
-    @IBOutlet var headingLabel: UILabel!
-    @IBOutlet var pitchLabel: UILabel!
-    @IBOutlet var rollLabel: UILabel!
+    @IBOutlet private var altitudeLabel: UILabel!
+    @IBOutlet private var headingLabel: UILabel!
+    @IBOutlet private var pitchLabel: UILabel!
+    @IBOutlet private var rollLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        updateUI()
+    }
+    
+    private var tableViewContentSizeObservation: NSKeyValueObservation?
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        tableViewContentSizeObservation = tableView.observe(\.contentSize) { [unowned self] (tableView, _) in
```

Comment:
I like seeing popovers sized to fit the enclosing table view just right. This is the best and easiest way I know how to do that. I suppose the proper height could be hard coded, but that only works when not using dynamic type.